perun-core/src/main/java/cz/metacentrum/perun/core/api/AuthzResolver.java

```diff
@@ -170,14 +178,14 @@ public static boolean hasRole(PerunPrincipal perunPrincipal, Role role) {
 	}
 
 	/**
-	 * Set role for user and all complementary objects.
+	 * Set role for user and <b>all</b> complementary objects.
 	 *
 	 * If some complementary object is wrong for the role, throw an exception.
 	 * For role "perunadmin" ignore complementary objects.
 	 *
 	 * @param sess perun session
 	 * @param user the user for setting role
-	 * @param role role 
+	 * @param role role of user in a session ( perunadmin | voadmin | groupadmin | self | facilityadmin | voobserver | topgroupcreator )
```

Comment:
You don't have to put full list of options here, since in java code, it's enum. It's reasonable for RPC javadoc only, where it's consumed as string.